quiche/src/h3/frame.rs

```diff
@@ -520,6 +525,14 @@ fn parse_settings_frame(
                 qpack_blocked_streams = Some(value);
             },
 
+            SETTINGS_H3_DATAGRAM_00 => {
```

Comment:
since the bodies are the same, consider collapsing this into:
```
SETTINGS_H3_DATAGRAM_00 | SETTINGS_H3_DATAGRAM =>
```

Reply:
SETTINGS_H3_DATAGRAM_00 | SETTINGS_H3_DATAGRAM =>

Reply:
11

Reply:
> since the bodies are the same, consider collapsing this into:
> ```
> SETTINGS_H3_DATAGRAM_00 | SETTINGS_H3_DATAGRAM =>
> ```

11